pkg/rest/request_test.go

```diff
@@ -48,26 +48,6 @@ func Test_deleteConfig(t *testing.T) {
 			http.StatusNotFound,
 			false,
 		},
-		{
```

Comment:
I guess this complete test can be removed (or possible moved to tests per client) - the current test setup will never return and error, as there's nothing that would cause a client error. 
Just testing that the test server & test client can communicate and return no error is pretty useless I'd say :D